x/concentrated-liquidity/pool_test.go

```diff
@@ -15,7 +15,10 @@ import (
 func (s *KeeperTestSuite) TestInitializePool() {
 	// Create a valid PoolI from a valid ConcentratedPoolExtension
 	validConcentratedPool := s.PrepareConcentratedPool()
-	validPoolI := validConcentratedPool.(poolmanagertypes.PoolI)
+	validPoolI, ok := validConcentratedPool.(poolmanagertypes.PoolI)
+	if !ok {
+		s.FailNow("validConcentratedPool is not a PoolI")
+	}
```

Comment:
nit: less verbosity improves readability which is preferable for tests IMO
```suggestion
	s.Require().True(ok)
```